requirements.txt

```diff
@@ -1,3 +1,4 @@
 # To edit the requirements file, open a PR on github.com/DataDog/datadog-agent-buildimages
 -r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements.txt
 semver==2.10.0
+codeowners==0.6.0
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏It'd be better to add it in buildimages. Also for `codeowners` it might be already installed in the pipeline at runtime, but you'd have to double check I might be wrong !

Reply:
I did remove this, but yes unfortunately I tried a test pipeline with this on friday and it failed: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/334810858